src/Exception/TestException.php

```diff
@@ -10,7 +10,7 @@
 /**
  * 
  */
-abstract class ExceptionAbstract extends \Exception
+abstract class TestException extends \Exception
```

Comment:
PSR defines that all interfaces end with the word "Interface" and all abstract classes should begin with the word "Abstract" so this class should be called `AbstractTestException`